remix.config.js

```diff
@@ -14,5 +14,6 @@ module.exports = {
     v2_meta: true,
     v2_normalizeFormMethod: true,
     v2_routeConvention: true,
+    unstable_tailwind: true, 
```

Comment:
You may consider to add in 
module.exports = {
  postcss: true,
  tailwind: true,
  // ...
It worked in my project.

Reply:
we have tested tailwind:true, but we've noticed tailwind don't work with only "tailwind: true," will experiment with adding "postcss: true"

Reply:
When I pulled this branch and added "tailwind:true", it worked for me.